test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetRestoreCommandTest.cs

```diff
@@ -327,6 +327,55 @@ public void RestoreCommand_FromSolutionFile(string configFileName)
             }
         }
 
+        [Theory]
+        [InlineData("packages.config")]
```

Comment:
Does this change work with packagesreference type projects? If yes then we may need another test for that one too.

Reply:
I don't think that's super relevant. 

This is about choosing which projects to restore which happens prior to any notion of packages.config vs PackageReference.